certmanager/README.md

```diff
@@ -11,3 +11,52 @@ in a standalone manner, separate from an SSL/TLS connection.
 $ make
 $ ./certverify
 ```
+## Verification of OQS Falcon Certificates
+
+The `generate_falcon_chains.sh` script will allow you to use the OQS project's
+OpenSSL in order to generate a self-signed CA certificate and entity
+certificate that use Falcon. In the OpenSSL directory, run the script to
+generate the certificates in the the /tmp/ directory.
+
+Apply the following patch:
+
+```
```

Comment:
Perhaps just create a new certverify_falcon.c example instead of suppling a patch. Also might be easier to use the local directory vs /tmp/?